docs/operations/data_migration.md

```diff
@@ -524,3 +524,31 @@ Example: `cf ssh getgov-za`
 | 2 | **debug**                  | Increases logging detail. Defaults to False.                                |
 | 3 | **limitParse**             | Determines how many domains to parse. Defaults to all.                      |
 | 4 | **disableIdempotentCheck** | Boolean that determines if we should check for idempotence or not. Compares the proposed extension date to the value in TransitionDomains. Defaults to False. |
+
+
+## Populate First Ready
```

Comment:
Agreed, though this would also entail moving some additional work in this file. May be good for another documentation related ticket because it also touches on some scripts that are lightly documented and ones that should be moved